### ISSUE TITLE
3.28.73.flags.json issue in 3.8.11

### ISSUE THREAD

**Issue by russback** [gulpjs/gulp#1051]
I've followed what others have done to fix this but nothing seems to work for me.

I've gone as far as removing node and npm altogether and installing again (Mac OS Mavericks) with Homebrew but I still get the issue.

* node version is 0.12.2
* npm version is 2.7.5
* local gulp version is 3.8.11
* global gulp version is 3.8.11

I've uninstalled local and global, run *npm cache clean* and reinstalled.

When I run *gulp* I get this error:

```
module.js:338
    throw err;
          ^
Error: Cannot find module '/Users/russback/npm/lib/node_modules/gulp/node_modules/v8flags/cache/3.28.73.flags.json'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/russback/npm/lib/node_modules/gulp/bin/gulp.js:25:22)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:501:10)
```

**Comment by phated** [comment]
@tkellen thoughts?

**Comment by tkellen** [comment]
@russback It doesn't seem like you're actually running `3.8.11`. The path being referenced in the stack trace is from the `1.x` series of v8flags, which was replaced back in 2014.

Uninstalling and re-installing node/npm doesn't remove your global `node_modules` folder. I'd try clearing out `~/npm/lib/node_modules` and re-installing gulp.

**Comment by phated** [comment]
This should be fixed with recent versions of gulp (3.8.11 or 3.9).  Please update completely.
